### ISSUE TITLE
Feature: add possibility to bypass plugin execution in pipeline simulation when ByPassPluginExecution=true is used

### ISSUE THREAD

**Issue by jordimontana82** [DynamicsValue/fake-xrm-easy#12]

**Comment by BetimBeja** [comment]
just putting a reference to the documentation here 😄 
https://learn.microsoft.com/en-us/power-apps/developer/data-platform/bypass-custom-business-logic?tabs=sdk#how-do-i-use-the-bypasscustompluginexecution-option

**Comment by jordimontana82** [comment]
@BetimBeja Indeed! And now we probably we need to look into the new preview feature as well!

https://learn.microsoft.com/en-us/power-apps/developer/data-platform/bypass-custom-business-logic-preview?tabs=sdk

**Comment by jordimontana82** [comment]
We'll look into these after #30 is merged first
